README.md

```diff
@@ -98,10 +97,10 @@ and Drop component to a lesson, then click the `EDIT` button.
 ![Edit view](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/edit-view.png)
```

Comment:
In the preview for this file, it embeds the current version of that image. The URL has a long-ish hash in it; once the repo is updated with the new image, will the existing link point to the new version of the image?

It looks like [Github allows relative in-repo links](https://help.github.com/articles/relative-links-in-readmes/), so we might want to go that way.